settings/src/script.js

```diff
@@ -163,9 +163,8 @@ function Main( { userId } ) {
 		);
 	} else if (
 		twoFactorRequiredScreens.includes( screen ) &&
-		userRecord.record[ '2fa_available_providers' ] &&
-		userRecord.record[ '2fa_revalidation' ] &&
-		userRecord.record[ '2fa_revalidation' ].expires_at <= new Date().getTime() / 1000
+		userRecord.record[ '2fa_available_providers' ].includes( 'Two_Factor_Totp' ) &&
+		userRecord.record[ '2fa_revalidation' ]?.expires_at <= new Date().getTime() / 1000
```

Comment:
If we're not going to disable the backup codes screen when totp is disabled, we'd need logic to check which provider is enabled, and show the relevant reauth UI; totp or backup code. Something like:

```
} else if (
		userRecord.record[ '2fa_available_providers' ].includes( 'Two_Factor_Totp' ) &&
		userRecord.record[ '2fa_revalidation' ]?.expires_at <= new Date().getTime() / 1000 
) {
                screenContent = (
			<>
				<div className={ 'wporg-2fa__account-status' }>
					<AccountStatus />
				</div>
				<RevalidateModalToTP />
			</>
		);
} else if (
		userRecord.record[ '2fa_available_providers' ].includes( 'Two_Factor_Backup_Codes' ) &&
		userRecord.record[ '2fa_revalidation' ]?.expires_at <= new Date().getTime() / 1000 
) {
                screenContent = (
			<>
				<div className={ 'wporg-2fa__account-status' }>
					<AccountStatus />
				</div>
				<RevalidateModalBackupCode />
			</>
		);
```